docs/README.md

```diff
@@ -1,540 +1,16 @@
 <!--readme-for-documentation-start-->
-## Kubestellar Website Overview 
+## KubeStellar Website Documentation
 
-### Websites
+**This section has been moved to a dedicated page for better organization and maintenance.**
 
-We have two web sites, as follows.
+For comprehensive information about contributing to the KubeStellar website, including:
+- Website overview and technology stack
+- How to preview changes (global and local)
+- Testing documentation PRs
+- Website structure and configuration
+- Publishing and versioning
+- Shortcut URLs
```

Comment:
This table of contents is already different from the one in the real document. I would not get into the business of maintaining two copies.